README.md

```diff
@@ -166,24 +160,37 @@ Expected Output: It will create .terraform directory in XC3/infrastructure/  loc
 7.  Please copy msg_templates in custodian directory on deployed EC2 instance
 
     ```
-    scp -i "keypair.pem" keypair.pem bastion-host-dns:/directory-to-copy-keypair
+    sudo scp -i "keypair.pem" keypair.pem bastion-host-dns:/directory-to-copy-keypair
```

Comment:
this keypair.pem is same as in the initial script?

Reply:
Yes, same as the initial script.
![image](https://github.com/XgridInc/xc3/assets/129849123/7b348441-cc46-444a-bd20-b8de9d48868c)